generate_changelog.py

```
@@ -0,0 +1,644 @@
+"""
+
+import json
+import subprocess
+import sys
+import re
+import argparse
+from datetime import datetime
+
+
+def run(cmd):
+    result = subprocess.run(cmd, shell=True, capture_output=True, text=True)
```

Ruff (0.15.21)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)